packages/SwingSet/test/snapshots/test-xsnap-store.js.md

```diff
@@ -19,16 +19,16 @@ Generated by [AVA](https://avajs.dev).
 
     {
       compressSeconds: 0,
-      hash: '3a62221548f1d4e9da4961e4fe1cff13f7170c9abfade9171eecfa58d6028c34',
+      hash: '06bf6499e476261317d8746465faf443e5c4a0a1b77f6b2c35f99dc3decd8a23',
```

Comment:
Let's see... why is this changing? I suppose because of things like...

```diff
-  `${src}\n//# sourceURL=xsnap/src/object-inspect.js\n`,
+  `${src}\n//# sourceURL=xsnap-lockdown/lib/object-inspect.js\n`,
```

I suppose every change in `lib/` affects this hash.

... which is why we're making this new package, I suppose.

Reply:
yep, exactly

Reply:
note: I don't know that the `sourceURL` will work, I just changed it to something that looked right. I haven't had a lot of experience/success with debuggers and source maps.